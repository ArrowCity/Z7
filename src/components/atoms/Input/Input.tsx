import React from "react";
import s from './style.less'

export interface InputProps {

}


export const Input : React.FC<InputProps> = () =>{
    return (
        <input className={s.demo}/>
    )
}
