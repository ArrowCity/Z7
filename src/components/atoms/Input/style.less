:local{
  .demo{
    width: 1000px;
  }
}
