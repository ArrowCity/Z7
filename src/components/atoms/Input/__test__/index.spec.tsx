import {mount} from 'enzyme'
import {Input} from "../Input";
import React from 'react';

describe('Button test --', function () {
    it('looks good', () => {
        const subject = mount(<Input/>)
        expect(subject).toMatchSnapshot();
    })
});
