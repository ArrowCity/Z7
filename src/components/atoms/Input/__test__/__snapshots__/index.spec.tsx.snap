// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test -- looks good 1`] = `
<Component>
  <input
    className="demo"
  />
</Component>
`;
